llvm/lib/Target/RISCV/RISCVInstrInfoVPseudos.td

```diff
@@ -527,11 +527,17 @@ class RISCVVPseudo {
   Instruction BaseInstr = !cast<Instruction>(PseudoToVInst<NAME>.VInst);
   // SEW = 0 is used to denote that the Pseudo is not SEW specific (or unknown).
   bits<8> SEW = 0;
+  bit IsNeeded = !cond(
+    !ne(!find(NAME, "PseudoVMCLR"), -1): 0,
```

Comment:
Can we set this from `VPseudoNullaryPseudoM`? Doing string matching here feels hacky.